### ISSUE TITLE
Milkway sometimes fully grays out the entire projection

### ISSUE THREAD

**Issue by j** [ofrohn/d3-celestial#18]
http://ofrohn.github.io/celestial-demo/viewer.html

Center: 0.8
h: -19.7

The entire canvas grays out.  How can we remove all the gray and have it be just the milkyway?

cc @ofrohn

**Comment by ofrohn** [comment]
Yes, I know about that issue. Whenever the path of the Milky Way is completely visible, it fills out the complete circle instead of just the MW-band. That is an internal issue with how D3 handles the path, and I haven't found a workaround yet, I'm sorry to say.
Any help with that would be welcome.

**Comment by j** [comment]
If I get any time, I can check it out.  It happens to do that at my location on my birthday! :P

**Comment by kiger** [comment]
Do you have some information about the problem? Maybe updating the d3 library solves it?

**Comment by Oceloty** [comment]
Hello! :) I also would like to kindly ask if there are any news to this issue or if anyone found a workaround, because I'm having problems with this bug in my current project, too. Thank you.

**Comment by ofrohn** [comment]
I think I found something! I made use of the fact that d3 paints the outside of polygons if the coordinate points are reversed in order, so after drawing the milky way I just paint the outside of the lowest layer in background-fill-color, darkening it in all orientations.
It turns out that layer also contained many dark clouds as interior polygons, those are now visible for the first time. So that went well. Check out the coal sack!
